core/include/gnuradio-4.0/Block.hpp

```diff
@@ -130,54 +133,65 @@ invokeProcessOneWithOrWithoutOffset(T &node, std::size_t offset, const Us &...in
         return node.processOne(inputs...);
 }
 
-template<std::size_t Index, typename Self>
+template<std::size_t Index, auto Kind, typename Self>
```

Comment:
could we constrain the `auto Kind` with a concept and/or use an explicite Type/NTTP for it?

Reply:
Ok